js/src/Ice/OutgoingAsync.js

```diff
@@ -456,7 +456,7 @@ export class OutgoingAsync extends ProxyOutgoingAsyncBase {
                     }
                 }
 
-                default:
+                default: {
```

Comment:
We should use `{}` when the case contains declarations. In this case we declare a const.